custom/icds_reports/migrations/sql_templates/database_views/poshan_progress_report_view.sql

```diff
@@ -0,0 +1,96 @@
+DROP VIEW IF EXISTS poshan_progress_report_view CASCADE;
+CREATE VIEW poshan_progress_report_view AS
+    SELECT
+        "awc_location_months"."district_id" AS "district_id",
+        "awc_location_months"."district_name" AS "district_name",
+        "awc_location_months"."state_id" AS "state_id",
+        "awc_location_months"."state_name" AS "state_name",
+        "awc_location_months"."month" AS "month",
+        "awc_location_months"."aggregation_level" AS "aggregation_level",
+        COALESCE("agg_awc"."num_launched_districts", 0) AS "num_launched_districts",
+        COALESCE("agg_awc"."num_launched_blocks", 0) AS "num_launched_blocks",
+        COALESCE("agg_awc"."num_launched_awcs", 0) AS "num_launched_awcs",
+        CASE WHEN COALESCE("agg_awc"."num_launched_awcs", 0)>0 THEN "agg_awc"."awc_days_open"/("agg_awc"."num_launched_awcs" * 25)::float*100 ELSE 0 END as "avg_days_awc_open_percent",
```

Comment:
could you calculate these percentages in python to be consistent with the existing pattern

Reply:
done